packages/firmware/src/screen.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+#include <LiquidCrystal.h>
+
+#define LCD_RS 2
+#define LCD_EN 3
+#define LCD_DB4 4
+#define LCD_DB5 5
+#define LCD_DB6 6
+#define LCD_DB7 7
+
+#define BAT_EMPTY (uint8_t)0x0
+#define BAT_ONE_QUARTERS (uint8_t)0x1
+#define BAT_TWO_QUARTERS (uint8_t)0x2
+#define BAT_THREE_QUARTERS (uint8_t)0x3
+#define BAT_FULL (uint8_t)0x4
+#define GPS_LOCK (uint8_t)0x5
+#define GPS_NO_LOCK (uint8_t)0x6
+
+static LiquidCrystal lcd(LCD_RS, LCD_EN, LCD_DB4, LCD_DB5, LCD_DB6, LCD_DB7);
+
+static uint8_t empty[8]          = {0x0E, 0x11, 0x11, 0x11, 0x11, 0x11, 0x1F};
```

Comment:
For some reason the compiler thinks these variables aren't used and warns about it. I'm not sure why, since it's used in `screen.cpp`.

Reply:
Is there any way to suppress the warning?

Reply:
I can probably turn of `-Wunused-variables`, but that's useful to have on an embedded system since there's a limited amount of memory available.

Reply:
Alright, probably just keep it then.